openedx_rocketchat/urls.py

```diff
@@ -0,0 +1,18 @@
+from .views import rocket_chat_discussion
+
+from django.conf import settings
+from django.conf.urls import url, include
+
+urlpatterns = [
+    url(
+        r'^api/',
+        include('openedx_rocketchat.api.urls', namespace='api'),
```

Comment:
Please, change to a better namespace name, like openedx-rocketchat-api @andrey-canon

Reply:
----